Lib/shutil.py

```diff
@@ -1242,8 +1242,8 @@ def get_terminal_size(fallback=(80, 24)):
     the value is a positive integer, it is used.
 
     When COLUMNS or LINES is not defined, which is the common case,
-    the terminal connected to sys.__stdout__ is queried
-    by invoking os.get_terminal_size.
+    the terminal connected to sys.__stdin__, sys.__stderr__, or sys.__stdout__
+    is queried by invoking os.get_terminal_size.
```

Comment:
I would prefer to trust ncurses developers and use the order that they suggested:
https://bugs.python.org/issue14841#msg323836

Please add a comment below on the loop iterating on streams, to add a reference to bpo-14841 to explain why in which order streams are tested.